lib/api/CInferenceModelDefinition.cc

```diff
@@ -857,6 +1057,20 @@ void CWeightedSum::addToDocument(rapidjson::Value& parentObject,
     writer.addMember(this->stringType(), object, parentObject);
 }
 
+void CWeightedSum::addToJsonStream(TGenericLineWriter& writer) const {
+    writer.StartObject();
+    writer.Key(this->stringType());
+    writer.StartObject();
+    writer.Key(JSON_WEIGHTS_TAG);
+    rapidjson::Value array{writer.makeArray(m_Weights.size())};
+    for (const auto& weight : m_Weights) {
+        array.PushBack(weight, writer.getRawAllocator());
+    }
+    writer.write(array);
```

Comment:
```suggestion
    writer.StartArray();
    for (const auto& weight : m_Weights) {
        writer.Double(weight);
    }
    writer.EndArray();
```

Reply:
It was a remainder of the first iteration. I use `addJsonArray()` now.